keps/1136-provisioning-request-support/README.md

```diff
@@ -193,6 +193,33 @@ type ProvisioningRequestConfigSpec struct {
 	// +optional
 	// +kubebuilder:default={backoffLimitCount:3,backoffBaseSeconds:60,backoffMaxSeconds:1800}
 	RetryStrategy *ProvisioningRequestRetryStrategy `json:"retryStrategy,omitempty"`
+
+	// podSetUpdates specifies the update of the workload's PodSetUpdates which
+	// are used to target the provisioned nodes.
+	//
+	// +optional
+	PodSetUpdates *ProvisioningRequestPodSetUpdates `json:"podSetUpdates,omitempty"`
+}
+
+type ProvisioningRequestPodSetUpdates struct {
+	// nodeSelector specifies the list of updates for the NodeSelector.
+	//
+	// +optional
+	NodeSelector []ProvisioningRequestPodSetUpdatesNodeSelector `json:"nodeSelector,omitempty"`
+}
+
+type ProvisioningRequestPodSetUpdatesNodeSelector struct {
```

Comment:
Why not just reuse the core API nodeSelector?

Reply:
Because this is just a specification how to build the selector. The value of the node selector is in the ProvisioningRequest status details, and is "randomly" generated by ClusterAutoscaler to be unique for the newly provisioned nodes.

For example in case of GKE, the node selector value will be under `ResizeRequestName: gke-default-job-sample-job-bf24c-f5a955a81bfd204e` ..., and the full selector we need to construct is: `autoscaling.gke.io/provisioning-request: gke-default-job-sample-job-bf24c-f5a955a81bfd204e`.

Reply:
That makes sense. Thank you.